blogs/posts/2024-01-16_nearest_neighbour.qmd

```diff
@@ -1,29 +1,38 @@
 ---
 title: "Nearest neighbour imputation"
-author: "Jacqueline Grout with Tom Jemmett"
+author:
+  - Jacqueline Grout
 date: "2024-01-16"
 categories: [learning]
 ---
 
-Recently I have been gathering data by GP practice, from a variety of different sources. The ultimate purpose of my project is to be able to report at an ICB/sub-ICB level. The various datasets cover different timescales and consequently changes in GP practices over time have left me with mismatching datasets.
+Recently I have been gathering data by GP practice, from a variety of different sources. The ultimate purpose of my project is to be able to report at an ICB/sub-ICB level (Integrated Care Board)\*. The various datasets cover different timescales and consequently changes in GP practices over time have left me with mismatching datasets.
```

Comment:
Could you convert this asterisk to [a Quarto footnote](https://quarto.org/docs/authoring/footnotes-and-citations.html#footnotes)? You can put '(Integrated Care Board)[^icb]' on this line and then below change '\\*An Integrated Care Board is...' to '[^icb]: An Integrated Care Board is...'

Reply:
Done - I think!